CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+- Fix-256: Add check for vagrant-registration plugin @budhrg
+- Refactored code for checking required plugins for CentOS files @budhrg
```

Comment:
@LalatenduMohanty Bex suggested me as it includes checking registration plugin for RHEL and required plugin for CentOS files